Implement ability to configure svgs ordering

Sprit3Dan opened the pull request:
As discussed in ticket #101, opening a PR containing chart/grid/decorators/extras ordering possibility implementation. Tested all stories on my side locally, order story added.

JesperLekland commented on src/chart.js:
```diff
@@ -165,8 +171,8 @@ Chart.defaultProps = {
-    renderDecorator: () => {
-    },
+    renderDecorator: () => {},
+    renderOrder: () => [ 'chart', 'grid', 'decorators', 'extras' ],
```
chart shouldn't be rendered under grid by default

Sprit3Dan commented:
Updated the PR correspondingly to your comments. Updated Readme, and changed default order.
Regarding implementation for all charts... Guess I'm not really into all the necessary details of the chart components, and if it's ok with, merge this PR to the baseline, and afterwards, let's think about your API proposal. I'm all yours for performing some refactoring :)

JesperLekland commented:
I think this solution is good enough as an intermediary step. I'll try to merge and release as soon as possible.
